central_service/app/frontend/app/templates/faqs/lambda-instance/find-master-node-hostname.hbs

```diff
@@ -0,0 +1,189 @@
+            </p>
+
+            <div class="row margin-bottom">
+              <div class="col-sm-6">
+                <div class="row margin-bottom">
+                  <div class="col-sm-3">
+                    <a href="#" data-toggle="modal" data-target="#LoginPage">
+                      <img class="img-responsive" src="assets/img/faqs/login-sm.png" alt="Login" title="Login" >
+                    </a>
+                    <p class="text-center">Step 1</p>
+                  </div>
+                  <div class="col-sm-1 text-center">
+                    <span class="fa fa-arrow-right"></span>
+                  </div>
+                  <div class="col-sm-3">
+                    <a href="#" data-toggle="modal" data-target="#createStep2">
+                      <img class="img-responsive" src="assets/img/faqs/lambda-instance/create-sm.png" alt="Home Page" title="Home Page">
+                    </a>
+                    <p class="text-center">Step 2</p>
+                  </div>
+                  <div class="col-sm-1 text-center">
+                    <span class="fa fa-arrow-right"></span>
+                  </div>
+                  <div class="col-sm-3">
+                    <a href="#" data-toggle="modal" data-target="#startstop">
+                      <img class="img-responsive" src="assets/img/faqs/lambda-instance/start-stop-sm.png" alt="Start Stop Lambda Instance" title="Start Stop Lambda Instance">
+                    </a>
+                    <p class="text-center">Step 3</p>
+                  </div>
+                </div>
+              </div><!--row-->
+              <div class="row ">
+                <div class="col-sm-7 col-md-offset-5">
+                  <span class="fa fa-arrow-down"></span>
+                </div>
+              </div>
+              <div class="row margin-bottom">
+                <div class="col-sm-6">
+                  <div class="row margin-bottom">
+                    <div class="col-sm-3 col-md-offset-8">
+                      <a href="#" data-toggle="modal" data-target="#Hostname">
+                        <img class="img-responsive" src="assets/img/faqs/lambda-instance/find-master-node-hostname/step4-sm.png" alt="Login" title="Login" >
+                      </a>
+                      <p class="text-center">Step 4</p>
+                    </div>
+                  </div>
+                </div><!--row-->
+              </div>
+            </div><!--row-->
+
+            <ol>
+              <li>
+                Log in the LoD Service VM using your ~okeanos token.
+              </li>
+
+              <li>
+                From the sidebar, select "Instances".
+              </li>
+
+              <li>
+                Click on the name of the Lambda Instance you chose or the corresponding "Details" button.
```

Comment:
Maybe it would be better to say 'Instance you wish' instead of 'Instance you chose'?